changelog.txt

```diff
@@ -1,10 +1,19 @@
 *** Changelog ***
 
-= 4.5.4 - 2020-xx-xx =
-* Add   - Security.md with security and vulnerability reporting guidelines.
+= 4.5.4 - 2020-11-16 =
+* Add   - Stripe Connect OAuth.
 * Tweak - Add site_url to all transactions, not just recurring ones.
 * Add   - Customer's full name is now included in Stripe Customer object if available.
 * Add   - Accept payments via AliPay when store currency is set to CNY.
+* Tweak - Wording of Apple Pay link.
+* Fix   - Serve Apple Pay domain verification file via a rewrite rule.
+* Add   - Apple Pay holiday marketing notice.
+* Fix   - Apple Pay: Ensure payer phone is available before attempting to access it.
+* Add   - Pass payment method for renewal charges if not a source.
+* Fix   - Ensure defaults are used for missing settings.
+* Add   - Add shipping address to payment requests.
+* Fix   - Fix the Not a valid URL notice.
+* Add   - Security.md with security and vulnerability reporting guidelines.
 
 = 4.5.3 - 2020-10-06 =
```

Comment:
I believe we were going to remove the 7th item here, which was a duplicate of the 4th one.

Reply:
Thanks for the reminder! Will do.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1370/commits/e9c0beb1279601be30bed09de9363ef92a4a7853